src/ReactWrapper.jsx

```diff
@@ -74,22 +75,29 @@ class ReactWrapper {
     }
 
     if (!root) {
+      this.renderer = getAdapter(options).createRenderer({ mode: 'mount', ...options });
       const ReactWrapperComponent = createWrapperComponent(nodes, options);
-      this.component = renderWithOptions(
+      this.renderer.render(
         <ReactWrapperComponent
```

Comment:
right, but isn't the point of adapters that the main enzyme package doesn't import React?